apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -121,8 +109,20 @@ interface CourseCodeCellProps {
 }
 
 const CourseCodeCell = withStyles(styles)((props: CourseCodeCellProps) => {
+    const appTheme = useThemeStore((store) => store.appTheme);
+
     const { classes, sectionCode } = props;
 
+    const [isHovered, setIsHovered] = useState(false);
+
+    const handleMouseEnter = () => {
```

Comment:
I think this slipped into your commit. It doesn't have to do with deprecating dark mode. 

You can delete the violating code and use `git commit--amend` to change the most recent commit.

Reply:
I added that code because the style on the Section Code Chip does not support the hover pseudo class, so I had to add my own hover event handling to implement the hover styles on the Section Code Chip. Is there a different way I can do this?